packages/server/api/routes/uidreview.router.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+// controllers
+const uidreviewController = require('../controllers/uidreview.controller');
+
+router.get('/:id/:uid', (req, res) => {
```

Comment:
I would put it like 
/:id/uid/:uid just to make it bit readable

Reply:
I have changed it to /:id/uid/:uid